tests/test_cfn_resource.py

```diff
@@ -115,11 +136,12 @@ def create(event, context):
     resp = rsrc(event, FakeLambdaContext())
 
     assert resp is None
-    assert urlmock.mock_calls == []
+    assert len(responses.calls) == 0
```

Comment:
afaik this isn't necessary as `responses.activate` without any registered response handlers means that requests fail with a `ConnectionError`